app/src/main/java/com/sun/hero_01/ui/SplashScreenActivity.kt

```diff
@@ -0,0 +1,14 @@
+package com.sun.hero_01.ui
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+class SplashScreenActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val intent = Intent(this, MainActivity::class.java)
+        startActivity(intent)
+        finish()
```

Comment:
đọc cái này nhé
https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md#25-arguments-in-fragments-and-activities